### PR TITLE
Edit the git config steps

### PR DESCRIPTION
**Pull request by cansavvy**
#### What changes are being implemented in this Pull Request?

There's been some issues with git config steps on personal repos: #628 

I've been doing some testing here: https://github.com/cansavvy/test-ottr/commit/6245c52391e7ba1f2121a4390bef67f3e5f4b1f5

This change made it work, I'm just wondering if it will be okay for org repos as well as wondering if we need it every time we make a commit?

### CODE REVIEW

**Comment by cansavvy** [comment]
The main change here is          
```
 git config --global --add safe.directory ${GITHUB_WORKSPACE}
git config --global user.email "itcrtrainingnetwork@gmail.com"
git config --global user.name "jhudsl-robot"
```
It is making things work See: https://github.com/fhdsl/AI_for_software/pull/8#event-8940313683

But @avahoffman I don't know why. Can you sanity check this?

**Comment by avahoffman** [comment]
@cansavvy have you tested doing away with the braces entirely?

e.g.:
```
git config --global --add safe.directory $GITHUB_WORKSPACE
git config --global user.email "itcrtrainingnetwork@gmail.com"
git config --global user.name "jhudsl-robot"
```

Also, I used --system and --global flags previously because sometimes the action didn't have permission to set global. No idea if this is still an issue.

**Comment by cansavvy** [comment]
@avahoffman I'll try it!

**Comment by cansavvy** [comment]
Oh it looks like you've done this already here: https://github.com/jhudsl/OTTR_Template/pull/633/commits/5f65ee99b614374d7e6dff6582d90163c6504b4c Looks like everything worked fine?

**Comment by avahoffman** [comment]
@cansavvy It worked here, but is there any concern that -no brackets- might not work in personal repos?

**Comment by cansavvy** [comment]
It *shouldn't* matter. I'm testing it here: https://github.com/cansavvy/test-ottr/pull/1 to make sure.

**Comment by cansavvy** [comment]
Appears to be working fine! This looks good to me, @avahoffman !

File: .github/workflows/pull_request.yml [hunk 1]
```diff
@@ -123,9 +124,9 @@ jobs:
       # Set up git checkout
       - name: Set up git checkout
         run: |
-          git config --system --add safe.directory "$GITHUB_WORKSPACE"
-          git config --local user.email "itcrtrainingnetwork@gmail.com"
-          git config --local user.name "jhudsl-robot"
+          git config --global --add safe.directory ${GITHUB_WORKSPACE}
```
Comment (avahoffman) [inline .github/workflows/pull_request.yml]:
```suggestion
          git config --global --add safe.directory $GITHUB_WORKSPACE
```

File: .github/workflows/render-all.yml [hunk 2]
```diff
@@ -49,9 +49,9 @@ jobs:
 
       - name: Login as jhudsl-robot
         run: |
-          git config --system --add safe.directory "$GITHUB_WORKSPACE"
-          git config --local user.email "itcrtrainingnetwork@gmail.com"
-          git config --local user.name "jhudsl-robot"
+          git config --global --add safe.directory ${GITHUB_WORKSPACE}
```
Comment (avahoffman) [inline .github/workflows/render-all.yml]:
```suggestion
          git config --global --add safe.directory $GITHUB_WORKSPACE
```

File: .github/workflows/send-updates.yml [hunk 3]
```diff
@@ -22,8 +22,9 @@ jobs:
 
       - name: Login as jhudsl-robot
         run: |
-          git config --local user.email "itcrtrainingnetwork@gmail.com"
-          git config --local user.name "jhudsl-robot"
+          git config --global --add safe.directory ${GITHUB_WORKSPACE}
```
Comment (avahoffman) [inline .github/workflows/send-updates.yml]:
```suggestion
          git config --global --add safe.directory $GITHUB_WORKSPACE
```

File: .github/workflows/starting-course.yml [hunk 4]
```diff
@@ -20,10 +20,10 @@ jobs:
 
       - name: Login as jhudsl-robot
         run: |
-          git config --system --add safe.directory "$GITHUB_WORKSPACE"
-          git config --local user.email "itcrtrainingnetwork@gmail.com"
-          git config --local user.name "jhudsl-robot"
-
+          git config --global --add safe.directory ${GITHUB_WORKSPACE}
```
Comment (avahoffman) [inline .github/workflows/starting-course.yml]:
```suggestion
          git config --global --add safe.directory $GITHUB_WORKSPACE
```

File: .github/workflows/test-send-updates.yml [hunk 5]
```diff
@@ -22,9 +22,9 @@ jobs:
 
       - name: Login as jhudsl-robot
         run: |
-          git config --system --add safe.directory "$GITHUB_WORKSPACE"
-          git config --local user.email "itcrtrainingnetwork@gmail.com"
-          git config --local user.name "jhudsl-robot"
+          git config --global --add safe.directory ${GITHUB_WORKSPACE}
```
Comment (avahoffman) [inline .github/workflows/test-send-updates.yml]:
```suggestion
          git config --global --add safe.directory $GITHUB_WORKSPACE
```

**Review by avahoffman — APPROVED** [review]
@cansavvy sounds good! Let's keep an eye on the `--global` flag just in case that breaks things 😬

**Comment by cansavvy** [comment]
If this passes, I will merge it.

**Pull request merged**